src/app.js

```diff
@@ -82,17 +77,13 @@ const app = function() {
       li.addEventListener('click', function(evt){
         evt.preventDefault();
 
-
-        if(!selectedCard) {
-            socket.emit('white card', card)
-            selectedCard = true;
-
-        }
-
+        // if(!selectedCard) {
```

Comment:
Removed this as it causes a bug that prevents anyone but card czar playing answers